samples: bluetooth: readme: miscellaneous sample doc fixes (#760)
nrfconnect/sdk-nrf-bm: samples/bluetooth/ble_lbs/README.rst

   You can configure the advertising name using the :kconfig:option:`CONFIG_SAMPLE_BLE_DEVICE_NAME` Kconfig option.
   For information on how to do this, see `Configuring Kconfig`_.
#. In nRF Connect for Desktop, scan for advertising devices.
   If the device is not advertising, reset the board with the :guilabel:`Reset Board` option in |VSC| or by pressing the reset button on the development kit.
#. :guilabel:`Connect` to your device.
   The terminal output in |VSC| indicates ``Peer connected``.
#. In nRF Connect for Desktop, go to :guilabel:`Nordic LED and Button Service` and change :guilabel:`Blinky LED state` to :guilabel:`01`.
   The terminal output in |VSC| indicates ``Received LED ON!``.
#. Change the :guilabel:`Blinky LED state` value back to :guilabel:`00`.
   The terminal output in |VSC| indicates ``Received LED OFF!``.
#. In the :guilabel:`Blinky Button State` characteristic, subscribe to button notifications by pressing the :guilabel:`Toggle notifications` button.

> #. Observe that the button state is notified when pressing **Button 2** on the kit.

Reviewer: ```suggestion
#. Observe that you receive a button state notification when pressing **Button 2** on the kit.
```

Author: Thanks!
